src/Playroom/SettingsPanel/SettingsPanel.tsx

```diff
@@ -73,101 +74,154 @@ const KeyboardShortcut = ({
+  }
+
+  const configTitle = window?.__playroomConfig__.title;
+
+  if (configTitle) {
+    return `${configTitle} | Playroom`;
+  }
+
+  return 'Playroom';
+};
+
 export default React.memo(() => {
-  const [{ editorPosition, colorScheme }, dispatch] = useContext(StoreContext);
+  const [{ editorPosition, colorScheme, localTitle }, dispatch] =
+    useContext(StoreContext);
 
   const keybindings = getKeyBindings();
 
+  const displayedTitle = getTitle(localTitle);
+
   return (
-    <ToolbarPanel data-testid="frame-panel">
-      <Stack space="large" dividers>
-        <fieldset className={styles.fieldset}>
-          <legend>
-            <Heading level="3">Editor Position</Heading>
-          </legend>
-          <div className={styles.radioContainer}>
-            {['Bottom', 'Right'].map((option) => (
-              <div key={option}>
-                <input
-                  type="radio"
-                  name="editorPosition"
-                  id={`editorPosition${option}`}
-                  value={option.toLowerCase()}
-                  title={option}
-                  checked={option.toLowerCase() === editorPosition}
-                  onChange={() =>
-                    dispatch({
-                      type: 'updateEditorPosition',
-                      payload: {
-                        position: option.toLowerCase() as EditorPosition,
-                      },
-                    })
-                  }
-                  className={styles.realRadio}
-                />
-                <label
-                  htmlFor={`editorPosition${option}`}
-                  className={styles.label}
-                  title={option}
-                >
-                  <span className={styles.labelText}>
-                    {positionIcon[option.toLowerCase() as EditorPosition]}
-                  </span>
-                </label>
+    <>
+      {localTitle === undefined ? null : (
+        <Helmet>
+          <title>{displayedTitle}</title>
+        </Helmet>
+      )}
+      <ToolbarPanel data-testid="frame-panel">
+        <Stack space="small" dividers>
+          <fieldset className={styles.fieldset}>
+            <legend>
+              <Heading level="3">Title</Heading>
+            </legend>
+            <input
+              type="text"
+              id="playroomTitleField"
+              placeholder="Enter a title for this Playroom..."
+              className={styles.textField}
+              value={localTitle}
+              onChange={(e) =>
+                dispatch({
+                  type: 'updateTitle',
+                  payload: { title: e.target.value },
+                })
+              }
+              onKeyDown={(e) => {
+                if (e.key === 'Enter') {
+                  dispatch({
+                    type: 'toggleToolbar',
+                    payload: { panel: 'settings' },
+                  });
+                }
+              }}
+            />
+          </fieldset>
```

Comment:
Given title is just one field, this should just be an input and label rather than a fieldset. We can run through this together separately.